bundles/org.eclipse.mita.program/src/org/eclipse/mita/program/generator/DefaultValueProvider.xtend

```diff
@@ -0,0 +1,192 @@
+package org.eclipse.mita.program.generator
```

Comment:
The dummy string generation seems to have an issue with strings in named constructor parameters:
<img width="436" alt="screen shot 2018-07-12 at 16 09 15" src="https://user-images.githubusercontent.com/3210701/42638484-0c9dd680-85ee-11e8-8412-4ed2dbd72e74.png">

Reply:
Yes, but only with platform things, there, the type doesn't resolve properly. In the past we didn't notice since the default was `"replace_me"` (with quotation marks). This is more correct IMHO.